index.js

```diff
@@ -196,12 +196,14 @@ module.exports = {
    * @returns {Array<String>} include paths
    */
   _getIncludesForDir: function(dir, prefix) {
+    const hasEmberCliTypescript = this.project && this.projectFindAddonByName && this.project.findAddonByName('ember-cli-typescript');
```

Comment:
this seems like a typo?

Shouldn't the second check be `this.project.findAddonByName`?

Reply:
@makepanic nice catch! Fixed

Reply:
Oops didn't see that you've already fixed it. Had the same issue nullvox initially reported and saw that in a diff between the other PR.